pkg/descheduler/strategies/pod_lifetime.go

```diff
@@ -50,13 +50,13 @@ func validatePodLifeTimeParams(params *api.StrategyParameters) error {
 // PodLifeTime evicts pods on nodes that were created more than strategy.Params.MaxPodLifeTimeSeconds seconds ago.
 func PodLifeTime(ctx context.Context, client clientset.Interface, strategy api.DeschedulerStrategy, nodes []*v1.Node, podEvictor *evictions.PodEvictor) {
 	if err := validatePodLifeTimeParams(strategy.Params); err != nil {
-		klog.V(1).Info(err)
+		klog.V(1).InfoS("err", err)
```

Comment:
should this have a message, followed by `"err", err` for the key=values? Right now this looks like you're passing InfoS with a message and a key, but no value. That may be allowed, but wanted to point it out...

Reply:
correct, but on second thought I think this error message could be left as is (before change). There's nothing to structure here that I can see anyway.

Reply:
or maybe we can add `strategy.Params.MaxPodLifeTimeSeconds`.

`klog.V(1).InfoS(err.Error(), "maxPodLifeTimeSeconds",  strategy.Params.MaxPodLifeTimeSeconds)`

`err.Error()` because `validatePodLifeTimeParams` returns different errors